css/navbar.css

```diff
@@ -0,0 +1,46 @@
+.navbar {
+    background-color: var(--color-black);
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    padding: 1rem 1.5rem;
+    border-bottom: 0.3px solid white;
```

Comment:
use color variable for white

Reply:
I have a color white variable(#EEEEEE)  for reuse, here I wanted white only(#FFFFFF) once so making a new variable didn't seem ideal. Thank you for the suggestion anyway.